### DISCUSSION TITLE
No LiDAR data for free, can't generate CDSM. Need advise to run SEBE

### DISCUSSION THREAD

**Discussion by sanvibol** [UMEP-dev/UMEP#426] [General]
Hello! I am new to GIS and UMEP, and I want to use SEBE/UMEP to estimate solar rooftop energy in Phnom Penh city, Cambodia. I could not download LiDAR data for free. I have DEM file and I use it to generate DSM but I don't have CDSM. How can I run SEBE without CDSM generating from LiDAR dataset? Please guide me how to do or if you have any documents for me to go through, please kindly share.

Thank you in advance!!

**Comment by biglimp** [comment]
You can run SEBE without CDSM (vegetation). If you dont have any data you can also generate using the [TreeGenerator](https://umep-docs.readthedocs.io/en/latest/pre-processor/Spatial%20Data%20Tree%20Generator.html). See end of this tutorial for inspiration (https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToSolweig.html#introtosolweig).
